fuseimg/data/ops/ops_debug.py

```diff
@@ -186,6 +188,105 @@ def __init__(
         self._show = show
         self._path = path
 
+    def frame_args(self, duration: int):
+        return {
+            "frame": {"duration": duration},
+            "mode": "immediate",
+            "fromcurrent": True,
+            "transition": {"duration": duration, "easing": "linear"},
+        }
+
+    def make_plotly_vis(self, vol=None):
+        """
+        args:
+            vol: lxwxh numpy-like matrix
+
+        output:
+           plotly.graph_objs._figure.Figure
+        """
+        # uncomment the below line for example from their website (brain MRI)
+        # vol = io.imread("<https://s3.amazonaws.com/assets.datacamp.com/blog_assets/attention-mri.tif>")
+        volume = vol.T
+        r, c = volume[0].shape
+
+        # Define frames
+        nb_frames = 68
+
+        fig = go.Figure(
+            frames=[
+                go.Frame(
+                    data=go.Surface(
+                        z=(6.7 - k * 0.1) * np.ones((r, c)), surfacecolor=np.flipud(volume[67 - k]), cmin=0, cmax=1
```

Comment:
(cmin, cmax) is the acceptable range of values that can be plotted. Plotly will interpolate values plotted to this range. Basically if all your pixels have value (0, 1) you do not want cmin and cmax to be (0, 255), because it will just look black.

I have simply set this to be the minimum and maximum of the sample itself. If there are any outlier/rogue pixels though then the data might not be visible. I can also expose these if you want but for most cases it should be fine the way I have implemented. 

Let me know if you want these to be exposed. Otherwise I set them like this:

```
cmin = vol.min()
cmax = vol.max()
```